querybook/server/lib/export/exporters/gspread_exporter.py

```diff
@@ -141,6 +145,13 @@ def _get_max_rows(
         )
         return max_result_rows - row_offset
 
+    def _save_sheet_to_statement_meta(self, sheet_url: str, statement_id: int):
+        statement_execution = get_statement_execution_by_id(statement_id)
+        meta_info = (
+            statement_execution.meta_info or ""
```

Comment:
does `statement_execution.meta_info` end with `\n`?

Reply:
looks like we add a `\n` at the end of every line for meta_info in querybook